### PR TITLE
Added configFile option

### PR DESCRIPTION
**Pull request by glook**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ const schema = {
         "Validate generated `*.d.ts` files and fail if an update is needed (useful in CI). Defaults to `false`",
       type: "boolean",
     },
+    configFile: {
```
Comment (Obi-Dann) [inline src/index.js:39]: I'd suggest calling it `prettierConfigFile` to make it clear that it has nothing to do with the loader config. Or perhaps extending `formatter` to support objects, eg:
```js
formatter: {
  type: 'prettier',
  configFile: './.prettierrc'
}
```

Comment (glook) [reply src/index.js:39]: Okay, im rename it and update pull request soon

Comment (Obi-Dann) [reply src/index.js:39]: Thanks! Really sorry about the delay, I added the comment ages ago but forgot to submit the review.

Comment (glook) [reply src/index.js]: @Obi-Dann done

**Review by Obi-Dann — APPROVED** [review]
LGTM! Thanks for the PR, going to create a release shortly!

**Pull request merged**